boards/arm/lpcxpresso55s36/lpcxpresso55s36_common.dtsi

```diff
@@ -0,0 +1,114 @@
+/*
```

Comment:
Do we need a `lpcxpresso55s36_common.dtsi` file. Seems like this can be move to `lpcxpresso55s36.dtsi`.

Reply:
I will try to move this file into lpcxpresso55s36.dtsi